package.json

```diff
@@ -25,15 +25,17 @@
     "eslint-plugin-react": "^7.14.2",
     "eslint-plugin-standard": "^4.0.0",
     "ethlint": "^1.2.4",
+    "openzeppelin-solidity": "2.0.1",
     "prettier": "^1.18.2",
-    "solidity-coverage": "^0.6.2"
+    "solidity-coverage": "^0.6.2",
+    "truffle": "^4.1.14"
```

Comment:
If you don't pin this specific version of truffle (by `"truffle": "4.1.14"`), this error arises when compiling contracts:
```
SyntaxError: Source file requires different compiler version (current compiler is 0.4.26+commit.4563c3fc.Emscripten.clang - note that nightly builds are considered to be strictly less than the released version
```

Reply:
Ahh i thought i had pinned it.